.github/workflows/trivy-scan.yml

```diff
@@ -0,0 +1,35 @@
+name: Trivy Docker Image Scan Workflow
+
+on:
+  schedule:
+    # 4am in UTC time.
+    - cron:  '0 4 * * *'
+  push:
+    branches:
+      - '*'
+    paths:
+      - './base/base-image/Dockerfile'
+  pull_request:
+    branches:
+      - '*'
+    paths:
+      - './base/base-image/Dockerfile'
+
+jobs:
+  build:
+    name: Trivy Docker Image Scan
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      - name: Set up Docker
+        uses: crazy-max/ghaction-docker-buildx@v1
+        with:
+          version: latest
+      - name: Install and run Trivy Scanning
+        run: |
+          export VERSION=$(curl --silent "https://api.github.com/repos/aquasecurity/trivy/releases/latest" | grep '"tag_name":' | sed -E 's/.*"v([^"]+)".*/\1/')
```

Comment:
Shouldn't we need to add `set -e`?

Reply:
It will fail without it. Example:
https://github.com/hemikak/docker/runs/545097223?check_suite_focus=true